Sergey_Zyryanov/src/components/ChatList/ChatList.jsx

```diff
@@ -0,0 +1,29 @@
+import React, {Component} from "react";
+import List from "@material-ui/core/List";
+import ListItem from "@material-ui/core/ListItem";
+import ListItemText from "@material-ui/core/ListItemText";
+
+import './ChatList.scss';
+
+export class ChatList extends Component {
+  render() {
+    return (
+      <div className='chat-list'>
```

Comment:
В аттрибутах лучше использовать двойные кавычки
```suggestion
      <div className="chat-list">
```